Jellyfin.Api/Controllers/MediaInfoController.cs

```diff
@@ -132,6 +133,7 @@ public async Task<ActionResult<PlaybackInfoResponse>> GetPlaybackInfo([FromRoute
         [FromQuery, ParameterObsolete] bool? enableTranscoding,
         [FromQuery, ParameterObsolete] bool? allowVideoStreamCopy,
         [FromQuery, ParameterObsolete] bool? allowAudioStreamCopy,
+        [FromQuery, ParameterObsolete] bool? alwaysBurnInSubtitleWhenTranscoding,
```

Comment:
Don't introduce new parameters that are immediately obsolete. This option should only come from the body.

Reply:
The problem: web is using deprecated apiclient for this endpoint

Reply:
Then web should use the newer non-deprecated endpoints via the SDK first. We're strictly not supporting legacy endpoints unless it's needed to keep a client working. In this case it's a new feature.